src/Native/Runtime/regdisplay.h

```diff
@@ -78,7 +78,7 @@ struct REGDISPLAY
     inline PCODE GetIP() { return IP; }
     inline PTR_PCODE GetAddrOfIP() { return pIP; }
     inline UIntNative GetSP() { return SP; }
-    inline UIntNative GetFP() { return *pR7; }
+    inline UIntNative GetFP() { return *pR11; }
```

Comment:
Could you please put this change under `PROJECTN` ifdef to avoid breaking .NET Native for UWP:

```
#ifdef PROJECTN
inline UIntNative GetFP() { return *pR7; }
#else
inline UIntNative GetFP() { return *pR11; }
#endif
```

Reply:
done